Makefile

```diff
@@ -186,6 +186,9 @@ watch-plugin: modd-precheck ## Run and upload the plugin to a development server
 live-watch-plugin: modd-precheck ## Run and update locally the plugin in the development server
 	cd mattermost-plugin; make live-watch
 
+watch-product: ## Run the product as something the Mattermost web app will watch for
```

Comment:
I tried to follow the naming scheme of the existing Makefile targets. Let me know if this works or if there's something better to use